<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>web</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>171</y>
     <width>611</width>
     <height>451</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>596</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>1000</height>
     </size>
    </property>
    <widget class="QLabel" name="label_word">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>591</width>
       <height>2000</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>得意黑</family>
       <pointsize>22</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_del" native="true">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>771</width>
     <height>161</height>
    </rect>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>761</width>
      <height>56</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_del">
       <property name="font">
        <font>
         <family>得意黑</family>
         <pointsize>22</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="text">
        <string>名称    </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_end_all">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>得意黑</family>
         <pointsize>22</pointsize>
         <italic>true</italic>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>761</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="button_end_search">
       <property name="font">
        <font>
         <family>得意黑</family>
         <pointsize>22</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>点击查询</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_end_change">
       <property name="font">
        <font>
         <family>得意黑</family>
         <pointsize>22</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>点击修改</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_end_del">
       <property name="font">
        <font>
         <family>得意黑</family>
         <pointsize>22</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>点击删除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_code">
       <property name="font">
        <font>
         <family>得意黑</family>
         <pointsize>22</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>序号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="line_code">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>得意黑</family>
         <pointsize>20</pointsize>
         <italic>true</italic>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_end_del_2">
       <property name="font">
        <font>
         <family>得意黑</family>
         <pointsize>22</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>确定删除</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>1</y>
     <width>201</width>
     <height>1001</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="button_exit">
      <property name="font">
       <font>
        <family>得意黑</family>
        <pointsize>22</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_add">
      <property name="font">
       <font>
        <family>得意黑</family>
        <pointsize>22</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>添加web</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_del">
      <property name="font">
       <font>
        <family>得意黑</family>
        <pointsize>22</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>删除web</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_search">
      <property name="font">
       <font>
        <family>得意黑</family>
        <pointsize>22</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>查询web</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_change">
      <property name="font">
       <font>
        <family>得意黑</family>
        <pointsize>22</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>修改web</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_add" native="true">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>621</y>
     <width>801</width>
     <height>371</height>
    </rect>
   </property>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>771</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_add1">
         <property name="font">
          <font>
           <family>得意黑</family>
           <pointsize>22</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="text">
          <string>名称    </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_add1">
         <property name="font">
          <font>
           <family>得意黑</family>
           <pointsize>22</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>名称</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_add2">
         <property name="font">
          <font>
           <family>得意黑</family>
           <pointsize>22</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>web   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_add2">
         <property name="font">
          <font>
           <family>得意黑</family>
           <pointsize>22</pointsize>
           <italic>true</italic>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>300</y>
      <width>511</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="button_add_2">
       <property name="font">
        <font>
         <family>得意黑</family>
         <pointsize>22</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>点击添加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_end_add">
       <property name="font">
        <font>
         <family>得意黑</family>
         <pointsize>22</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>继续添加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_end_change_2">
       <property name="font">
        <font>
         <family>得意黑</family>
         <pointsize>22</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>修改</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
